sandbox/.gitignore

```diff
@@ -0,0 +1,4 @@
+**/*
```

Comment:
Can't we do this at the top level?

Reply:
I wanted to keep it isolated to the sandbox directory. But no strong objections to move it to top.